go-controller/pkg/ovn/policy.go

```diff
@@ -93,6 +96,57 @@ func (oc *Controller) syncNetworkPolicies(networkPolicies []interface{}) {
 	}
 }
 
+func (oc *Controller) getNetworkPoliciesLocked() map[types.NamespacedName]*networkPolicy {
+	//Get lock on controller's networkPolicies struct
+	oc.networkPoliciesMutex.Lock()
+
+	npInfo := oc.networkPolicies
```

Comment:
:thinking: can you return oc.networkPolicies directly?

Reply:
I believe so, in golang Slices, maps and channels are reference types that do not require the extra indirection of a pointer